src/helpers/params.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2024  Study association GEWIS
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ *  @license
+ */
+
+/**
+ * Converts SQL with :named parameters and a params object
+ * to SQL with positional (?) placeholders and a matching array.
+ * Handles each parameter only in the order it appears in the SQL.
+ */
+export function convertToPositional(sql: string, params: Record<string, any>): { sql: string; values: any[] } {
```

Comment:
Any way to do this without `any`? That should prevent tech debt in the future.

_(For ParelPracht, most of the issues I have to solve are related to the use of `any`...)_